subsys/caf/modules/power_manager.c

```diff
@@ -307,6 +322,9 @@ static bool event_handler(const struct event_header *eh)
 	return false;
 }
 EVENT_LISTENER(MODULE, event_handler);
+#if IS_ENABLED(CONFIG_CAF_FORCE_POWER_DOWN_EVENTS)
+	EVENT_SUBSCRIBE(MODULE, force_power_down_event);
+#endif
```

Comment:
if keep alive event can be enabled/disabled we should check it here too

Reply:
Done